Documentation/op-guide/gateway.md

```diff
@@ -63,4 +63,12 @@ Start the etcd gateway to fetch the endpoints from the DNS SRV entries with the
 ```bash
 $ etcd gateway --discovery-srv=example.com
 2016-08-16 11:21:18.867350 I | tcpproxy: ready to proxy client requests to [...]
-```
\ No newline at end of file
+```
+
+## Changing listener address
+
+By default, etcd gateway listens for client requests on `127.0.0.1:23790`. This can be changed using `--listen-addr`, eg:
+
+```bash
+$ etcd gateway start --endpoints=infra0.example.com --listen-addr=127.0.0.1:2222
```

Comment:
`etcd gateway start --endpoints=infra0.example.com:2379 --listen-addr=127.0.0.1:2379`

so the remote endpoint has a port (otherwise fails to dial with "missing port in address") and local listen port matches the default etcd port?